tests/e2e/statefulsets.go

```diff
@@ -422,6 +424,151 @@ var _ = ginkgo.Describe("[csi-block-vanilla] [csi-supervisor] statefulset", func
 		ssPodsAfterScaleDown = fss.GetPodList(client, statefulset)
 		gomega.Expect(len(ssPodsAfterScaleDown.Items) == int(replicas)).To(gomega.BeTrue(), "Number of Pods in the statefulset should match with number of replicas")
 	})
+
+	/*
+		verify online volume expansion on statefulset
+			1. Create a SC with allowVolumeExpansion set to 'true' in SVC
+			2. create stateful set with replica 3 using the above created SC
+			3. Once all the statefull set PODs are up follow the below step to edit stateful set
+			4. kubectl edit pvc <pvcName>  for each PVC in the StatefulSet, to increase its capacity.
+			5. kubectl delete sts --cascade=false <statefullSetName>  to delete the StatefulSet and leave its pods.
+			6. vi statefulset.yaml and edit the storage and increase the size to the size you have edited the PVC in step4
+			7. create the same stateful set again
+			8. Scaleup statefulset
+			9. Newly created stateful set should have the increased size
+			10. scale down statefulset to 0
+			11. delete statefulset and all PVC's and SC's
+	*/
+	ginkgo.It("[csi-supervisor] verify online volume expansion on statefulset", func() {
+		ctx, cancel := context.WithCancel(context.Background())
+		defer cancel()
+		var pvcSizeBeforeExpansion int64
+
+		ginkgo.By("Creating StorageClass for Statefulset")
+		// decide which test setup is available to run
```

Comment:
Test is for csi-supervisor and the comment says decide which test setup is available - which is a contradictory 
Please correct